packages/webapp/pages/onboarding.tsx

```diff
@@ -187,7 +189,11 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.ContentTypes);
     }
 
-    if (activeScreen === OnboardingStep.ContentTypes && isMobile) {
+    if (
+      activeScreen === OnboardingStep.ContentTypes &&
+      isMobile &&
+      isPushSupported
```

Comment:
We need to either fix the export of `isPushSupported` or do this:
https://github.com/dailydotdev/apps/commit/15eed06762d267a923b56637b1ae73a192102666

(Best is first advice)

Just try it in Brave browser it blocks OneSignal, so you can see if it works or not.

Reply:
I will fix it from the root to avoid any issues down the line 👀